src/porting_android.h

```diff
@@ -97,9 +97,7 @@ int getInputDialogSelection();
 
 bool hasPhysicalKeyboardAndroid();
 
-#ifndef SERVER
 float getDisplayDensity();
 v2u32 getDisplaySize();
-#endif
```

Comment:
Why can this be removed?
Do we not support android server builds?

Reply:
we don't really support them